examples/file_apy_example.py

```diff
@@ -0,0 +1,119 @@
+"""
+example.py
+
+Example script to demonstrate the usage of the CS3Client class.
+Start with an empty directory and you should end up with a directory structure like this:
+
+test_directory1
+test_directory2
+test_directory3
+rename_file.txt (containing "Hello World")
+text_file.txt
+"""
+
+import logging
+import configparser
+from cs3client import CS3Client
+from cs3resource import Resource
+
+config = configparser.ConfigParser()
+with open("default.conf") as fdef:
+    config.read_file(fdef)
+# log
+log = logging.getLogger(__name__)
+
+client = CS3Client(config, log)
```

Comment:
Don't you need a config category?

Reply:
I do, I thought I added this while testing, must've removed it when changing something.